### PR TITLE
#34 - Add GPL license and improve README

### PR DESCRIPTION
**Pull request by teto**

### CODE REVIEW

**Comment by teto** [comment]
Guiss I am unlucky :(
```
[1052/1960] Compiling src/mesh/model/dot11s/hwmp-protocol.cc
g++: internal compiler error: Killed (program cc1plus)
Please submit a full bug report,
with preprocessed source if appropriate.
See <file:///usr/share/doc/gcc-4.9/README.Bugs> for instructions.
```

**Comment by thehajime** [comment]
> Guiss I am unlucky :(

I rerun the test: will see.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+Development is done on github and bugs should be filed at https://github.com/direct-code-execution/ns-3-dce/issues.
+
+The mailing list used to discuss internals is the same as ns3's: ns-developers at isi.edu
+
+
+
+
+License
+===
+DCE is available under the GPLv3 license. A copy is available in the repository.
```
Comment (thehajime) [inline README.md]: no, it's GPLv2, following ns-3.

**Comment by teto** [comment]
I've updated with GPLv2.
(I've just a last patch concerning netlink export to pcap I would like to push before an official release.)

**Comment by teto** [comment]
Can I merge this ? it sounds pretty uncontroversial to me while being of importance.

**Comment by thehajime** [comment]
Merged, thank.

**Pull request merged**